### PR TITLE
fix #1218

### LINKED ISSUE #1218: Error when accessing some cases
for instance this one: https://scout-stage.scilifelab.se/cust003/18069

Error is the following:

File "/home/hiseq.clinical/miniconda/envs/prod-stage/lib/python3.6/site-packages/flask_bootstrap/templates/bootstrap/base.html", line 1, in top-level template code
    {% block doc -%}
  File "/home/hiseq.clinical/miniconda/envs/prod-stage/lib/python3.6/site-packages/flask_bootstrap/templates/bootstrap/base.html", line 4, in block "doc"
    {%- block html %}
  File "/home/hiseq.clinical/miniconda/envs/prod-stage/lib/python3.6/site-packages/flask_bootstrap/templates/bootstrap/base.html", line 20, in block "html"
    {% block body -%}
  File "/home/hiseq.clinical/miniconda/envs/prod-stage/lib/python3.6/site-packages/flask_bootstrap/templates/bootstrap/base.html", line 23, in block "body"
    {% block content -%}
  File "/home/hiseq.clinical/SERVER/apps/scout-stage/git/scout-stage/scout/server/blueprints/cases/templates/cases/case.html", line 60, in block "content"
    {{ comments_panel(institute, case, current_user, comments) }}
  File "/home/hiseq.clinical/miniconda/envs/prod-stage/lib/python3.6/site-packages/jinja2/runtime.py", line 579, in _invoke
    rv = self._func(*arguments)
  File "/home/hiseq.clinical/SERVER/apps/scout-stage/git/scout-stage/scout/server/templates/utils.html", line 68, in template
    {% if comment.created_at < case.updated_at %}
TypeError: '<' not supported between instances of 'datetime.datetime' and 'list'

Looks like either comment.created or case.updated is a list and not a datetime object! Thanks @barrystokman for noticing!

### PR DESCRIPTION
**Pull request by northwestwitch**
Check if case.updated_at is a list. If it is, set it to the latest updated datetime.

I haven't changed the code in utils.html, just indented it to be more clear!

### CODE REVIEW

**Comment by northwestwitch** [comment]
testing this on stage!

**Comment by northwestwitch** [comment]
Works on stage: https://scout-stage.scilifelab.se/cust003/18069

File: scout/server/blueprints/cases/controllers.py [hunk 1]
```diff
@@ -145,6 +145,10 @@ def case(store, institute_obj, case_obj):
 
     case_obj['clinvar_variants'] = store.case_to_clinVars(case_obj['_id'])
 
+    # if updated_at is a list, set it to the last update datetime
+    if case_obj.get('updated_at') and isinstance(case_obj['updated_at'], list):
```
Comment (dnil) [inline scout/server/blueprints/cases/controllers.py]: :+1:

Comment (northwestwitch) [reply scout/server/blueprints/cases/controllers.py]: Thanks!

**Pull request merged**